doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -60,6 +60,26 @@ The issues in this section are related to :ref:`protocols`.
 Amazon Sidewalk
 ===============
 
+.. rst-class:: v2-6-0
+
+KRKNWK-18465: BUS fault on FSK during the FACTORY_RESET call
+  The Bluetooth LE link is deinitialized.
+  However, a race condition might occur where an event added by the Bletooth LE link is not removed.
```

Comment:
```suggestion
  However, a race condition might occur where an event added by the Bluetooth® LE link is not removed.
```

Reply:
Also, could remove the trademark from line no: 87 and make the above as "*Bluetooth®*"